### PR TITLE
additions to streamtube doc

### PR DESCRIPTION
**Pull request by emmanuelle**
I hope this is the right place for this PR. I had a hard time understanding how streamtube works, especially the divergence encoding the radius was only documented in a community post I think.

### CODE REVIEW

File: src/traces/streamtube/index.js [hunk 1]
```diff
@@ -53,7 +53,9 @@ module.exports = {
             'By default, the tubes\' starting positions will be cut from the vector field\'s',
             'x-z plane at its minimum y value.',
             'To specify your own starting position, use attributes `starts.x`, `starts.y`',
-            'and `starts.z`.'
+            'and `starts.z`.',
+	    'The color is encoded by the norm of (u, v, w), and the local radius',
+	    'by the divergence of (u, v, w)'.
```
Comment (antoinerg) [inline src/traces/streamtube/index.js]: `'.` should be `.'`

Comment (emmanuelle) [reply src/traces/streamtube/index.js]: oops! no wonder the CI failed!

**Comment by etpinard** [comment]
:dancer: - nice improvement!

If ever you're interested in diving deeper into the streamtube algorithm, this is where it happens:

https://github.com/gl-vis/gl-streamtube3d/blob/master/streamtube.js

Thanks very much!

**Comment by emmanuelle** [comment]
Thank you @etpinard , merging.

**Pull request merged**